src/consumer/assigners/roundRobinAssigner/index.js

```diff
@@ -42,20 +41,24 @@ module.exports = ({ cluster }) => ({
     const sortedMembers = members.map(({ memberId }) => memberId).sort()
     const assignment = {}
 
-    sortedMembers.forEach(memberId => {
-      assignment[memberId] = {}
+    const topicsPartionArrays = topics.map(topic => {
+      const partitionMetadata = cluster.findTopicPartitionMetadata(topic)
+      return partitionMetadata.map(m => ({ topic: topic, partitionId: m.partitionId }))
     })
+    const topicsPartitions = [].concat(...topicsPartionArrays)
```

Comment:
Everything looks good, can you use `utils/flatten` instead of `[].concat(...)`?

Reply:
Gladly!

Reply:
Done